maestro-cli/src/main/java/maestro/cli/command/PrintHierarchyCommand.kt

```diff
@@ -81,6 +81,13 @@ class PrintHierarchyCommand : Runnable {
     )
     private var compact: Boolean = false
 
+    @CommandLine.Option(
+        names = ["--device-index"],
+        description = ["The index of the device to run the test on"],
+        hidden = true
+    )
+    private var deviceIndex: Int? = null
+
```

Comment:
Do we just need this on hierarchy command?

Reply:
`test` uses the first device, `start-device` requires a device. So I'm guessing so, yes